examples/elasticsearch-incremental/README.md

```diff
@@ -0,0 +1,243 @@
+# Elasticsearch
+
+[Elasticsearch](https://www.elastic.co/) Elasticsearch is a search engine based 
+on the Lucene library. It provides a distributed, multitenant-capable full-text
+ search engine with an HTTP web interface and schema-free JSON documents.
+
+This blueprint use the 
+[elasticsearch snapshot api](https://www.elastic.co/guide/en/elasticsearch/reference/current/snapshot-restore.html) 
+to protect your cluster. This snapshot is incremental and the blueprint works 
+only for 
+[s3 compatible repository](https://www.elastic.co/guide/en/elasticsearch/reference/current/repository-s3.html).
+
+## Prerequisites
+
+* Kubernetes 1.9+
+* PV support on the underlying infrastructure
+* Kanister controller version 0.81.0 or above installed in your cluster
+* Kanctl CLI installed (https://docs.kanister.io/tooling.html#kanctl)
+
+## Operator Details
+
+We will be using the 
+[elasticsearch operator](https://www.elastic.co/guide/en/cloud-on-k8s/current/index.html) 
+to deploy the elasticsearch cluster.
+
+## Installing the Operator and the cluster
+
+Install the operator
+```
+kubectl create -f https://download.elastic.co/downloads/eck/2.4.0/crds.yaml
+kubectl apply -f https://download.elastic.co/downloads/eck/2.4.0/operator.yaml
+```
+
+Monitor the logs 
+```
+kubectl -n elastic-system logs -f statefulset.apps/elastic-operator
+```
+
+Create a cluster in test-es1 namespace
```

Comment:
should we describe this a bit

```suggestion
Create an Elastic search cluster in `test-es1` namespace by creating `Elasticsearch` resource
```

Reply:
yes that would avoid confusion around the word "cluster"